app.yaml

```diff
@@ -14,79 +14,60 @@ inbound_services:
 - warmup
 - mail
 
+default_expiration: "10m"
 handlers:
+# PRODUCTION STATIC
 - url: /favicon.ico
   static_files: assets/favicon.ico
   upload: assets/favicon.ico
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /robots.txt
   static_files: assets/robots.txt
   upload: assets/robots.txt
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /sitemap.xml
   static_files: assets/sitemap.xml
   upload: assets/sitemap.xml
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /build
   static_dir: build
   secure: always
-  expiration: 30d
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
+  expiration: "90d"
+- url: /third_party/static
+  static_dir: third_party/static
+  secure: always
+  expiration: "90d"
+- url: /mapreduce/pipeline/images
```

Comment:
Maybe move this higher (with /sitemap.xml) so that stuff with the same cache policy is roughly in the same place.

Reply:
Done.